examples/multistrandtest.py

```diff
@@ -36,7 +36,7 @@ def multiColorWipe(color1, color2, wait_ms=5):
         if i % 2:
             # even number
             strip1.setPixelColor(i, color1)
-            strip2.setPixelColor(i / 2, color2)
+            strip2.setPixelColor(int(i / 2), color2)
```

Comment:
```suggestion
            strip2.setPixelColor(i // 2, color2)
```
Integer division is the cleaner way to do this, same effect.

Reply:
Didnt know about Integer division, thanks! From a quick test this seems to do the trick.